doc/source/tutorials/quickstart.rst

```diff
@@ -357,84 +374,239 @@ Step 6: Now to make it turn
+            
+        }.
+        
+    }.
+
+    PRINT "100km apoapsis reached, cutting throttle".
+
+    //At this point, our apoapsis is above 100km and our main loop has ended. Next
+    //we'll make sure our throttle is zero and that we're pointed prograde
+    LOCK THROTTLE TO 0.
+
+    //This sets the user's throttle setting to zero to prevent the throttle
+    //from returning to the position it was at before the script was run.
+    SET SHIP:CONTROL:PILOTMAINTHROTTLE TO 0.
 
-    // NOTE that it is vital to not just let the script end right away
-    // here.  Once a kOS script just ends, it releases all the controls
-    // back to manual piloting so that you can fly the ship by hand again.
-    // If the program just ended here, then that would cause the throttle
-    // to turn back off again right away and nothing would happen.
 
 And here is it in action:
 
-.. figure:: /_images/tutorials/quickstart/example_2_5.png
+.. figure:: /_images/tutorials/quickstart/example_2_6.png
     :width: 80 %
 
 And toward the end:
 
-.. figure:: /_images/tutorials/quickstart/example_2_6.png
+.. figure:: /_images/tutorials/quickstart/example_2_7.png
     :width: 80 %
 
-If we assume you made a vessel that has enough fuel and power to get up to orbit, this script should in principle work to get you to the point of leaving the atmosphere. It will probably still fall back down, because this script makes no attempt to ensure that the craft is going fast enough to maintain the orbit.
+This script should, in principle, work to get you to the point of leaving the atmosphere. It will probably still fall back down, because this script makes no attempt to ensure that the craft is going fast enough to maintain the orbit.
 
 As you can probably see, it would still have a long way to go before it would become a really GOOD launching autopilot. Think about the following features you could add yourself as you become more familiar with **kOS**:
 
-- You could change the steering logic to make a more smooth gravity turn by constantly adjusting the pitch in the HEADING according to some math formula. The example shown here tends to create a "too high" launch that's a bit inefficient.
-- You could complete the launching script by making sure once the vessel breaks the atmosphere it actually makes a circular orbit rather than just stopping after 70000m and coasting.
-- This script just stupidly leaves the throttle at max the whole way. You could make it more sophisticated by adjusting the throttle as necessary to avoid too much wasted energy fighting air friction. (The way **KSP**'s stock areodynamic model works, the optimal speed is terminal velocity, by the way). This is partly addressed in the :ref:`PID Loop Tutorial <pidloops>`.
-- With more sophisticated staging checks, the script could be made to work with solid fuel engines as well.
-- With even more sophisticated checks, the script could be made to work with fancy staging methods like asaparagus.
```

Comment:
I'd put this bullet point back in.  The MAXTHRUST = 0 check does not do asparagus staging properly, so that is still an improvement users can make.  (With asparagus staging, maxthrust doesn't go all the way down to zero when it's time to stage - it drops, but it doesn't drop all the way to zero, because you have some engines burned out but other engines still going.)